channel/channels/channel_instances.py

```diff
@@ -94,6 +108,6 @@ def del_chan_at_id(chan_name, chan_id) -> None:
     try:
         ChannelInstances.instance().channels[chan_id].pop(chan_name, None)
     except KeyError:
-        get_logger(ChannelInstances.__name__).warning(
+        logging.getLogger(ChannelInstances.__name__).warning(
```

Comment:
We will need a way to log into octobot loggers when available

Reply:
Yes

Reply:
We can probably add a method `getLogger` in octobot commons and try to replace logging from default logging to commons one